packages/vega-functions/src/codegen.js

```diff
@@ -274,6 +275,7 @@ expressionFunction('treeAncestors', treeAncestors, dataVisitor);
 
 // register Vega-Lite selection functions
 expressionFunction('vlSelectionTest', selectionTest, selectionVisitor);
+expressionFunction('vlComparisonTest', comparisonTest, selectionVisitor);
```

Comment:
The name `comparisonTest` is rather generic, and breaks the convention of naming selection-specific methods with the `selection` prefix. Can we name it something like `selectionCompare` instead? The registered expression function name would then be `vlSelectionCompare`.